spec/controllers/restaurants_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+RSpec.describe RestaurantsController, type: :controller do
+  describe "actions" do
+    describe "GET index" do
+      let!(:restaurants) do
+        FactoryBot.create_list(:restaurant, 2)
+      end
+
+      before(:each) do
+        get(:index)
+      end
+
+      it "checking count" do
```

Comment:
the "it" statement inside should have a more meaningful statement:

`it "restaurants response should be equal to 2" do`